libraries/SuplaDevice/examples/Supla_RollerShutter_FRAM/Supla_RollerShutter_FRAM.ino

```diff
@@ -161,6 +166,18 @@ void setup() {
   
   SuplaDevice.setRollerShutterFuncImpl(&supla_rs_SavePosition, &supla_rs_LoadPosition, &supla_rs_SaveSettings, &supla_rs_LoadSettings);
   
+  
+  /* If you are using secure connections over SSL (ESP8266 boards only),and you know the supla-server certificate fingerprint
+   * uncomment following lines and fill proper server's certificate thumbprint
+   * Remember if certificate would change you will need to change that value if uncommented
+   * If you leave it commented the certificate will not be validated but connection will be encrypted
+   */ 
+  
+  //char fingerprint[MAX_FINGERPRINT_SIZE] = "9b:a8:18:b9:5e:d6:06:52:f8:22:15:00:e1:56:68:8d:7a:61:11:77";
+  //wifi.setFingerprint(fingerprint);
+  
```

Comment:
maybe you could add one new example file with fingerprint set? Then don't modify all existing examples, because there are already too many options in comments.